components/wifi/esp8266-driver/ESP8266Interface.h

```diff
@@ -403,6 +412,11 @@ class ESP8266Interface : public NetworkStack, public WiFiInterface {
         mbed::DigitalOut  _pwr_pin;
     } _pwr_pin;
 
+    /** Assert the reset and power pins
+     *    ESP8266 has two pins serving similar purpose and this function asserts them both if they are configured in mbed_app.json.
```

Comment:
Indentation.

Reply:
You're right, I guess astyle config is taking advantage of that _preferably_ keyword :D

> Each line preferably has at most 120 characters.

Reply:
I guess so too :D As we know opinions start to differ once you go beyond 80 characters.